src/main/java/seedu/duke/data/notebook/Notebook.java

```diff
@@ -68,6 +204,46 @@ public void setNotes(ArrayList<Note> notes) {
         this.notes = notes;
     }
 
+    public String archiveNotes(int index) {
+        Note archivedNote = notes.get(index);
+
+        archivedNotes.add(archivedNote);
+        deleteNote(index);
```

Comment:
just do notes.remove(index) here
```suggestion
        notes.remove(index);
```